CogniteSdk.Types/Alpha/Simulators/SimulatorModelRevisionData.cs

```diff
@@ -0,0 +1,297 @@
+// Copyright 2024 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+#nullable enable
```

Comment:
this is the only place in the SDK we user `#nullable enable`, let's check with someone, maybe @dbrattli if it's a good idea?